### PR TITLE
Bug 2027896: [release-4.8] Add support for Azure Marketplace Images

### PR DESCRIPTION
**Pull request by patrickdillon**
Vendors new API field for Azure Marketplace Images

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,38 +5,38 @@ go 1.13
...
-	k8s.io/kubectl v0.21.0
-	k8s.io/utils v0.0.0-20210722164352-7f3ee0f31471
+	k8s.io/api v0.22.1
+	k8s.io/apimachinery v0.22.1
+	k8s.io/apiserver v0.22.0
+	k8s.io/client-go v0.22.1
+	k8s.io/code-generator v0.22.1
+	k8s.io/klog/v2 v2.10.0
+	k8s.io/kubectl v0.22.0
+	k8s.io/utils v0.0.0-20210819203725-bdf08cb9a70a
```
Comment (JoelSpeed) [inline go.mod:32]: Not sure we should be bumping the K8s dependencies to 1.22 on the 4.8 branch, why did this get changed?

Comment (patrickdillon) [reply go.mod:32]: I must have replaced at `@master` instead of the intended `@release-4.8`. Fixing now.

**Comment by JoelSpeed** [comment]
/override ci/prow/e2e-libvirt 
/override ci/prow/e2e-aws-disruptive

The libvirt tests are permanently broken and should not be required. Equally, the disruptive tests are currently broken across the board. The failures are not related to this change

**Pull request merged**